demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -75,13 +76,60 @@ def match(_dict: dict, path: Path) -> bool:
     def pack_id(self) -> str:
         return self.in_pack.pack_id if self.in_pack else ""
 
+    @validator("pack", always=True)
+    def validate_pack(cls, v: Any, values) -> Optional["Pack"]:
+        # Validate that we have the pack containing the content item.
+        # The pack is either provided directly or needs to be located.
+
+        pack = cls.get_pack(values.get("relationships_data"), values.get("path"))
+        if v and not isinstance(v, fields.FieldInfo):
```

Comment:
when will it not be FieldInfo?

Reply:
Great question, It was copied from what Ilan wrote here:
https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/content_graph/objects/content_item.py#L132

I usually avoid changing things that seem to be there on purpose, even if I don't get why. But I tried removing it and the build was ok with it. What do you think?

Reply:
I think we should check if it's necessary and remove if not
it's just confusing if it's not relevant and next time people do something similar they will add it also for no reason